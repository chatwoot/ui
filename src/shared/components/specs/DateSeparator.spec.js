import { mount } from '@vue/test-utils';
import DateSeparator from '../DateSeparator';

describe('DateSeparator', () => {
  test('matches snapshot', () => {
    const wrapper = mount(DateSeparator, {
      propsData: {
        date: 'Nov 18, 2019',
      },
      mocks: {
        $t: () => {},
      },
    });
    expect(wrapper.vm).toBeTruthy();
    expect(wrapper.element).toMatchSnapshot();
  });
});
