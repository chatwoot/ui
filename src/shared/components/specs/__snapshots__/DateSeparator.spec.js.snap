// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DateSeparator matches snapshot 1`] = `
<div
  class="date--separator"
>
  
  Nov 18, 2019

</div>
`;
